examples/sim/qrdqn_monolith_discrete_sim.py

```diff
@@ -0,0 +1,247 @@
+import pprint
+import sys
+sys.path.append("..")
+
+import numpy as np
+import torch
+from examples import DQN,QRDQN
+from examples import wrap_offworld, wrap_offworld_real
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.data import Collector, VectorReplayBuffer
+from tianshou.env import SubprocVectorEnv 
+from tianshou.policy import QRDQNPolicy
+from tianshou.trainer import offpolicy_trainer
+from tianshou.utils import TensorboardLogger
+
+import gym
+import offworld_gym
+from offworld_gym.envs.common.channels import Channels
+from offworld_gym.envs.common.enums import AlgorithmMode, LearningType
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) # to surpress the warning when running real env
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--task', type=str, default='OffWorldDockerMonolithDiscreteSim-v0')
+    parser.add_argument('--seed', type=int, default=0)
+    parser.add_argument('--eps-test', type=float, default=0.005)
+    parser.add_argument('--eps-train', type=float, default=1.)
+    parser.add_argument('--eps-train-final', type=float, default=0.1)
+    parser.add_argument('--buffer-size', type=int, default=25000)
+    parser.add_argument('--lr', type=float, default=0.0003)
+    parser.add_argument('--gamma', type=float, default=0.95)
+    parser.add_argument('--num-quantiles', type=int, default=200)
+    parser.add_argument('--n-step', type=int, default=3)
+    parser.add_argument('--target-update-freq', type=int, default=200)
+    parser.add_argument('--epoch', type=int, default=250)
+    parser.add_argument('--step-per-epoch', type=int, default=1000)
+    parser.add_argument('--step-per-collect', type=int, default=100)
+    parser.add_argument('--update-per-step', type=float, default= 0.1)
+    parser.add_argument('--batch-size', type=int, default=256)
+    parser.add_argument('--training-num', type=int, default=1)
+    parser.add_argument('--test-num', type=int, default=1)
+    parser.add_argument('--logdir', type=str, default='log')
+    parser.add_argument('--render', type=float, default=0.)
+    parser.add_argument(
+        '--device', type=str, default='cuda' if torch.cuda.is_available() else 'cpu'
+    )
+    parser.add_argument('--frames-stack', type=int, default=1)
+    parser.add_argument('--resume-path', type=str, default=None)
+    parser.add_argument(
+        '--watch',
+        default=False,
+        action='store_true',
+        help='watch the play of pre-trained policy only'
+    )
+    parser.add_argument('--save_buffer_name', type=str, default="buffer.hdf5")
+    parser.add_argument(
+        "--load_buffer_name", type=str, default="buffer.hdf5"
+    )
+    parser.add_argument("--save-interval", type=int, default=4)
+    parser.add_argument('--resume', action="store_true")
+    return parser.parse_args()
+
+
+def make_offworld_env(args):
+    return wrap_offworld(args.task)
+
+
+def make_offworld_env_watch(args):
```

Comment:
Can this be also removed and streamlined in the code?

Reply:
Think function just calls another function, can be refactored to just call `wrap_offworld` instead of `make_offworld_env_watch`

Reply:
Sure